clients/python-wrapper/tests/utests/test_auth_flow.py

```diff
@@ -0,0 +1,361 @@
+import datetime
+from unittest.mock import MagicMock, patch
```

Comment:
I would much rather prefer to continue using monkeypatching than introducing another unit test library to the code.